app/src/org/commcare/tasks/FormRecordCleanupTask.java

```diff
@@ -345,9 +345,7 @@ private static void wipeRecord(Context context, int sessionId,
                             "Inconsistent formRecordId's in session storage");
                 }
             } catch (Exception e) {
-                Logger.exception(e);
-                Logger.log(LogTypes.TYPE_ERROR_ASSERTION,
-                        "Session ID exists, but with no record (or broken record)");
+                Logger.exception("Session ID exists, but with no record (or broken record)", e);

```

Comment:
I feel like ideally we should be  tagging such logs with both `TYPE_ERROR_ASSERTION` and `exception` for such cases. Is it possible to provide multiple tags to a log with SUMO ?

Reply:
I haven't currently set it up to make that possible, no. we'd have to just send the same log twice with different tags